frontend/src/screens/channels/IncreaseIncomingCapacity.tsx

```diff
@@ -203,246 +212,265 @@ function NewChannelInternal({
+                                    Min. channel size:
+                                  </span>
+                                  {new Intl.NumberFormat().format(
+                                    selectedPeer.minimumChannelSize
+                                  )}{" "}
+                                  sats
+                                </span>
+                              </div>
+                            </div>
+                            <Sheet
+                              open={ChannelPartnersMenuOpen}
+                              onOpenChange={setChannelPartnersMenuOpen}
+                            >
+                              <SheetTrigger>
+                                <Button
+                                  variant="secondary"
+                                  type="button"
+                                  className="flex gap-2 items-center justify-center"
+                                >
+                                  <RefreshCw className="w-4 h-4 mr-2" />
+                                  Change
+                                </Button>
+                              </SheetTrigger>
+                              <SheetContent>
+                                <SheetHeader>
+                                  <SheetTitle>Change channel peer</SheetTitle>
+                                  <SheetDescription>
+                                    <div className="grid gap-4">
+                                      {channelPeerSuggestions
+                                        .filter(
+                                          (peer) =>
+                                            peer.network === network &&
+                                            peer.paymentMethod ===
+                                              order.paymentMethod
+                                        )
+                                        .map((peer) => (
+                                          <Card
+                                            key={getPeerKey(peer)}
+                                            className={`p-4 shadow-none cursor-pointer ${
+                                              getPeerKey(selectedPeer) ===
+                                              getPeerKey(peer)
+                                                ? "border-primary"
+                                                : ""
+                                            }`}
+                                            onClick={() =>
+                                              setSelectedPeer(peer)
+                                            }
+                                          >
+                                            <div className="flex items-center gap-3 justify-between">
+                                              <div className="flex items-center gap-3">
+                                                {peer.name !== "Custom" && (
```

Comment:
There is no custom any more. If you make big changes to these flows can you consider what needs to be removed and what needs to be kept?

Reply:
resolved